README.md

```diff
@@ -190,6 +195,13 @@ to its corresponding digest. See the different
 [pull specifications](https://github.com/containerbuildsystem/operator-manifest#pull-specifications)
 to which this process applies to.
 
+## Gating Bundle Images
+
+In addition to building operator index images, IIB can also gate your bundle images before adding
+them to the index image. If Greenwave configuration is setup for your queue, IIB will query
```

Comment:
It might also be nice to link Greenwave here to the docs as the above comment mentioned.
```suggestion
them to the index image. If a Greenwave configuration is setup for your queue, IIB will query
```